express-notes-api/index.js

```diff
@@ -0,0 +1,106 @@
+const express = require('express');
+const app = express();
+const data = require('./data.json');
+const notes = data.notes;
+const jsonMiddleware = express.json();
+const fs = require('fs');
+app.use(jsonMiddleware);
+
+app.get('/api/notes', (req, res) => {
+  const notesArray = Object.values(notes);
+  res.status(202);
+  res.json(notesArray);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const id = Number(req.params.id);
+  const errorMessage = { error: 'id must be a positive integer' };
+  const notFound = { error: `cannot find note with id ${id}` };
+  if (id <= 0 || isNaN(id)) {
```

Comment:
I would recommend using thing for the `integer` check: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger